### ISSUE TITLE
Accessibility: dates should be marked with the `<time>` tag

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#154]
The dates are currently not properly formatted with a `<time>` tag, which makes the readout less than ideal.
